sbt-scalajs-bundler/src/main/scala/scalajsbundler/JSDOMNodeJSEnv.scala

```diff
@@ -46,45 +46,89 @@ class JSDOMNodeJSEnv(
   protected trait AbstractDOMNodeRunner extends AbstractNodeRunner {
 
     protected def codeWithJSDOMContext(): Seq[VirtualJSFile] = {
-      val scriptsJSPaths = getLibJSFiles().map {
-        case file: FileVirtualFile => file.path
-        case file                  => libCache.materialize(file).getAbsolutePath
+      val scriptsFiles = (getLibJSFiles() :+ code).map {
```

Comment:
Why do we need this change?

Reply:
I just copy the changes from https://github.com/scala-js/scala-js/commit/8d1938b6eaa985c933189f0eaf82edee0b3cb897#diff-b097042b8f58e84b044e481289fa803d